docs/administration/garbage-collection/_index.md

```diff
@@ -38,10 +36,10 @@ To avoid triggering the garbage collection process too frequently, the availabil
     * **Daily**: Run garbage collection at midnight every day.
     * **Weekly**: Run garbage collection at midnight every Saturday.
     * **Custom**: Run garbage collection according to a `cron` job.
-    
-1. To delete untagged artifacts, select the check box **Delete Untagged Artifacts**.     
+
+1. To **Allow garbage collection on untagged artifacts**, select the option. If you have this option turned on, the next time Garbage collect runs on your Harbor instance, Harbor will delete untagged artifacts and then perform garbage collection on them.     
 1. Click **Save**.
-1. Select the **History** tab to view records of the 10 most recent garbage collection runs.
+1. View the 10 most recent garbage collection runs in the **GC History** table.
```

Comment:
now, it's 5, not 10.

Reply:
Thanks! I updated this to be more generic so we dont have to worry about specific numbers if this changes in the future!